conda_forge_tick/make_graph.py

```diff
@@ -57,8 +57,9 @@ def get_attrs(name, i, bad):
         source = [source]
     source_keys = set()
     for s in source:
-        if not sub_graph.get('url'):
-            sub_graph['url'] = s.get('url')
+        if s.get('url'):
+            sub_graph['url'] = s['url']
+            break
```

Comment:
Ah is that the problem spot?

Reply:
Nope, that's actually a mistake. I thought this might help but then realized it would break other things and forgot to change it back.